### ISSUE TITLE
Cannot reorder tabs while in private windows. New windows are created instead.

### ISSUE THREAD

**Issue by noslenac** [brave/brave-browser#6264]
## Description 

On Ubuntu Linux 18.04, while in a private tab, tab reordering does not work as expected. Instead of changing the tab order, it tears off into a new window. There is also no way to move a private tab back into the private window it was torn off.

## Steps to Reproduce

   1. Open a new private window
   2. Create some tabs
   3. Attempt to reorder
   4. Attempt to merge windows

## Actual result:

- Cannot merge private windows
- Cannot reorder tabs in a private window

## Expected result:
- Should be able to reorder tabs in a private window
- Should be able to move tab back into another private window.

## Reproduces how often: 

Easily reproduced

## Brave version (brave://version info)

Brave | 0.68.142 Chromium: 77.0.3865.90 (Official Build) (64-bit)
Revision | 58c425ba843df2918d9d4b409331972646c393dd-refs/branch-heads/3865@{#830}
OS | Linux

**Comment by bsclifton** [comment]
cc: @srirambv @btlechowski (are you able to reproduce?)

**Comment by btlechowski** [comment]
Not able to reproduce.


Brave | 0.68.142 Chromium: 77.0.3865.90 (Official Build) (64-bit)
-- | --
Revision | 58c425ba843df2918d9d4b409331972646c393dd-refs/branch-heads/3865@{#830}
OS | Ubuntu 18.04 LTS

I can move the tabs around without a problem. Also can torn the tab off to new window and merger it back.

![6264](https://user-images.githubusercontent.com/34715963/66139609-04b77b00-e601-11e9-93c5-76317c5266a4.gif)

**Comment by noslenac** [comment]
I removed and reinstalled brave on my primary Linux computer and it seems it is acting appropriately. 

I will leave a private window open with multiple tabs all day to verify it is not time related. If I cannot reproduce the issue, I'll close the issue.

**Comment by noslenac** [comment]
Purging Brave from all of my Linux computers seems to have corrected this issue, so it seems this was caused by an upgrade between versions at some point.
